packages/block-editor/src/components/font-sizes/fluid-utils.js

```diff
@@ -51,7 +49,6 @@ export function getComputedFluidTypographyValue( {
 	minimumViewPortWidth = DEFAULT_MINIMUM_VIEWPORT_WIDTH,
 	maximumViewPortWidth = DEFAULT_MAXIMUM_VIEWPORT_WIDTH,
 	scaleFactor = DEFAULT_SCALE_FACTOR,
-	minimumFontSizeFactor = DEFAULT_MINIMUM_FONT_SIZE_FACTOR,
```

Comment:
Will removing this parameter cause any back compat issues, given that `getComputedFluidTypographyValue` is part of the public block editor API?

Reply:
Thanks for raising that. Yeah, it's possible. 

The `minimumFontSizeFactor` was a value only configurable in the JS function `getComputedFluidTypographyValue()`

In the PHP, the size factor was hardcoded, and was never an option that could be passed: https://github.com/WordPress/gutenberg/pull/49707/files#diff-2c54d3c8305590cf826f83511a9fd85d5b405470addd62edc3183ff9118177fbL486

In that way, this PR harmonizes the front and backend code 😅 I guess the question is whether there are JS usages out there where one would expect it work as before (in the editor at least). 

But my guess is that it'd be relatively safe given that the backend code never allowed customization of this value.

Furthermore, I've removed `minimumFontSizeFactor` from the JS completely in this PR so passing it won't cause an error. 

I can't find any explicit usages on https://wpdirectory.net/, but happy to follow folks' advice if there are backwards compat alarm bells. 

Besides this, in general, this PR does represent a change in the way the min values are calculated and therefore has the potential to change the way a site's typography behaves. Also it changes the default min viewport, which will do the same.